### ISSUE TITLE
Error when add too many nuclides  to a material

### ISSUE THREAD

**Issue by D-Benjamin** [openmc-dev/openmc#1362]
Hi All,
    There is error when I add many nuclides to a material like that:
```
fuelSalt1 = openmc.Material(0, "core")
fuelSalt1.set_density('g/cm3', 4.123)
fuelSalt1.add_nuclide('Li6', 1.135000e+01)
fuelSalt1.add_nuclide('Li7', 4.702000e+06)
fuelSalt1.add_nuclide('F19', 2.752000e+07)
fuelSalt1.add_nuclide('Th232', 3.686000e+07)
fuelSalt1.add_nuclide('U233', 5.025000e+06)
fuelSalt1.add_nuclide('Th230', 8.374000e+02)
fuelSalt1.add_nuclide('Pa233', 2.425000e+04)
fuelSalt1.add_nuclide('U232', 4.684000e+02)
fuelSalt1.add_nuclide('U234', 2.217000e+06)
fuelSalt1.add_nuclide('U235', 5.092000e+05)
fuelSalt1.add_nuclide('U236', 3.646000e+05)
fuelSalt1.add_nuclide('U237', 4.293000e+02)
fuelSalt1.add_nuclide('U238', 2.872000e+02)
fuelSalt1.add_nuclide('Np237', 7.521000e+04)
fuelSalt1.add_nuclide('Pu238', 6.278000e+04)
fuelSalt1.add_nuclide('Pu239', 1.898000e+04)
fuelSalt1.add_nuclide('Pu240', 7.248000e+03)
fuelSalt1.add_nuclide('Pu241', 1.059000e+03)
fuelSalt1.add_nuclide('Pu242', 2.738000e+02)
fuelSalt1.add_nuclide('Am241', 1.665000e+02)
fuelSalt1.add_nuclide('Am243', 4.781000e+01)
fuelSalt1.add_nuclide('Sr88', 1.970000e+03)
fuelSalt1.add_nuclide('Sr89', 9.961000e+02)
fuelSalt1.add_nuclide('Sr90', 4.124000e+04)
fuelSalt1.add_nuclide('Y91', 2.053000e+03)
fuelSalt1.add_nuclide('Sn126', 1.370000e+03)
fuelSalt1.add_nuclide('Te130', 1.346000e+03)
fuelSalt1.add_nuclide('Cs137', 1.821000e+04)
fuelSalt1.add_nuclide('Ba136', 1.794000e+03)
fuelSalt1.add_nuclide('Ba137', 1.379000e+03)
fuelSalt1.add_nuclide('Ba138', 2.577000e+04)
fuelSalt1.add_nuclide('Ba140', 1.826000e+03)
fuelSalt1.add_nuclide('La139', 3.703000e+03)
fuelSalt1.add_nuclide('Ce140', 4.737000e+03)
fuelSalt1.add_nuclide('Ce141', 2.903000e+03)
fuelSalt1.add_nuclide('Ce142', 5.663000e+03)
fuelSalt1.add_nuclide('Ce143', 1.999000e+02)
fuelSalt1.add_nuclide('Ce144', 3.812000e+03)
```
But when I add any one more nuclide, a error will happen like that:
```
terminate called after throwing an instance of 'std::out_of_range'
  what():  _Map_base::at
Aborted (core dumped)

```
I find it can only add no more than 39 nuclides. So, Is it because that openmc can not deal with so many nuclides? Any reply will appreciated!

**Comment by D-Benjamin** [comment]
> What version of OpenMC are you running? I just tried this with v0.10.0 on the develop branch and had no issues.

Hi, can you add some more nuclides and try again? Attach like that:
```
fuelSalt1.add_nuclide('Pr143', 1.352000e+03)
fuelSalt1.add_nuclide('Nd143', 3.655000e+03)
fuelSalt1.add_nuclide('Nd145', 3.050000e+03)
fuelSalt1.add_nuclide('Nd146', 2.418000e+03)
fuelSalt1.add_nuclide('Nd148', 1.272000e+03)
```
By the way, my version is v0.11.0 on the develop branch(latest version). Thank you

**Comment by gridley** [comment]
Yep, indeed, I tried adding those 5 extra and it worked. Hm. People have depletion working with a couple hundred nuclides so this seems to be a problem relating to your installation.

**Comment by D-Benjamin** [comment]
> Yep, indeed, I tried adding those 5 extra and it worked. Hm. People have depletion working with a couple hundred nuclides so this seems to be a problem relating to your installation, maybe some new commit broke stuff!

Thank you very much, I reckon there is a bug with new version

**Comment by D-Benjamin** [comment]
I may find the reason, if I use DAGMC-OpenMC, it will occur those error, but it's ok when use OpenMC directly.

**Comment by pshriwise** [comment]
Sounds like this is entering my territory. I'll have a look and try to reproduce that behavior.

**Comment by pshriwise** [comment]
Odd, I just tried adding 45 nuclides to one of the materials in our regression tests, but I didn't see this error.

**Comment by D-Benjamin** [comment]
Thank u, there is a limit of adding nuclides, tomorrow I upload my script.  now it's midnight in our country, and I'll go to sleep

**Comment by D-Benjamin** [comment]
> Odd, I just tried adding 45 nuclides to one of the materials in our regression tests, but I didn't see this error.

Hi psdriwise, 
here is my script. And I also find that if I change the material id from "0" to another integer, it will not report this error. But if I retain this id=0, the nuclides adding is limited to 60 nuclides.

```
#!/usr/bin/python3
# -*- coding: utf-8 -*-

import os
import openmc

# define materials
fuelSalt1 = openmc.Material(0, "core")
fuelSalt1.set_density('g/cm3', 4.123)
fuelSalt1.add_nuclide('Li6', 1.135000e+01)
fuelSalt1.add_nuclide('Li7', 4.702000e+06)
fuelSalt1.add_nuclide('F19', 2.752000e+07)
fuelSalt1.add_nuclide('Th229', 1.117000e+02)
fuelSalt1.add_nuclide('Th230', 8.374000e+02)
fuelSalt1.add_nuclide('Th232', 3.686000e+07)
fuelSalt1.add_nuclide('U233', 5.025000e+06)
fuelSalt1.add_nuclide('Pa233', 2.425000e+04)
fuelSalt1.add_nuclide('U232', 4.684000e+02)
fuelSalt1.add_nuclide('U234', 2.217000e+06)
fuelSalt1.add_nuclide('U235', 5.092000e+05)
fuelSalt1.add_nuclide('U236', 3.646000e+05)
fuelSalt1.add_nuclide('U237', 4.293000e+02)
fuelSalt1.add_nuclide('U238', 2.872000e+02)
fuelSalt1.add_nuclide('Np237', 7.521000e+04)
fuelSalt1.add_nuclide('Np238', 1.028000e+02)
fuelSalt1.add_nuclide('Pu238', 6.278000e+04)
fuelSalt1.add_nuclide('Pu239', 1.898000e+04)
fuelSalt1.add_nuclide('Pu240', 7.248000e+03)
fuelSalt1.add_nuclide('Pu241', 1.059000e+03)
fuelSalt1.add_nuclide('Pu242', 2.738000e+02)
fuelSalt1.add_nuclide('Am241', 1.665000e+02)
fuelSalt1.add_nuclide('Am243', 4.781000e+01)
fuelSalt1.add_nuclide('Se80', 1.346000e+02)
fuelSalt1.add_nuclide('Se82', 3.156000e+02)
fuelSalt1.add_nuclide('Br81', 1.580000e+02)
fuelSalt1.add_nuclide('Rb87', 4.307000e+02)
fuelSalt1.add_nuclide('Sr88', 1.970000e+03)
fuelSalt1.add_nuclide('Sr89', 9.961000e+02)
fuelSalt1.add_nuclide('Sr90', 4.124000e+04)
fuelSalt1.add_nuclide('Y89', 6.833000e+02)
fuelSalt1.add_nuclide('Y91', 2.053000e+03)
fuelSalt1.add_nuclide('Sn122', 1.896000e+02)
fuelSalt1.add_nuclide('Sn123', 1.902000e+01)
fuelSalt1.add_nuclide('Sn124', 4.198000e+02)
fuelSalt1.add_nuclide('Sn125', 1.887000e+01)
fuelSalt1.add_nuclide('Sn126', 1.370000e+03)
fuelSalt1.add_nuclide('Te128', 2.939000e+02)
fuelSalt1.add_nuclide('Te130', 1.346000e+03)
fuelSalt1.add_nuclide('Te132', 2.355000e+02)
fuelSalt1.add_nuclide('I129', 2.220000e+02)
fuelSalt1.add_nuclide('I131', 2.346000e+02)
fuelSalt1.add_nuclide('Cs135', 3.680000e+02)
fuelSalt1.add_nuclide('Cs137', 1.821000e+04)
fuelSalt1.add_nuclide('Ba136', 1.794000e+03)
fuelSalt1.add_nuclide('Ba137', 1.379000e+03)
fuelSalt1.add_nuclide('Ba138', 2.577000e+04)
fuelSalt1.add_nuclide('Ba140', 1.826000e+03)
fuelSalt1.add_nuclide('La139', 3.703000e+03)
fuelSalt1.add_nuclide('La140', 2.292000e+02)
fuelSalt1.add_nuclide('Ce140', 4.737000e+03)
fuelSalt1.add_nuclide('Ce141', 2.903000e+03)
fuelSalt1.add_nuclide('Ce142', 5.663000e+03)
fuelSalt1.add_nuclide('Ce143', 1.999000e+02)
fuelSalt1.add_nuclide('Ce144', 3.812000e+03)
fuelSalt1.add_nuclide('Pr141', 3.099000e+03)
fuelSalt1.add_nuclide('Pr143', 1.352000e+03)
fuelSalt1.add_nuclide('Nd143', 3.655000e+03)
fuelSalt1.add_nuclide('Nd144', 4.887000e+02)
fuelSalt1.add_nuclide('Nd145', 3.050000e+03)    #60th nuclides
fuelSalt1.add_nuclide('Nd146', 2.418000e+03)    #61th nuclides
# fuelSalt1.add_nuclide('Nd147', 4.336000e+02)
# fuelSalt1.add_nuclide('Nd148', 1.272000e+03)
# fuelSalt1.add_nuclide('Nd150', 5.141000e+02)
# fuelSalt1.add_nuclide('Pm147', 1.287000e+03)
# fuelSalt1.add_nuclide('Sm149', 6.939000e+02)
# fuelSalt1.add_nuclide('Sm151', 3.002000e+02)
# fuelSalt1.add_nuclide('Sm152', 2.377000e+02)
# fuelSalt1.add_nuclide('Eu153', 1.208000e+02)
fuelSalt1.add_s_alpha_beta('c_Li_in_LiF')
fuelSalt1.add_s_alpha_beta('c_F_in_LiF')
fuelSalt1.temperature = 975

fuelSalt2 = openmc.Material(1001, "top_pipe")
fuelSalt2.set_density('g/cm3', 4.0825)
fuelSalt2.add_nuclide('Li6', 1.135000e+01)
fuelSalt2.add_nuclide('Li7', 4.702000e+06)
fuelSalt2.add_nuclide('F19', 2.752000e+07)
fuelSalt2.add_nuclide('Th232', 3.686000e+07)
fuelSalt2.add_nuclide('U233', 5.025000e+06)
fuelSalt2.add_nuclide('Th230', 8.374000e+02)
fuelSalt2.add_nuclide('Pa233', 2.425000e+04)
fuelSalt2.add_nuclide('U232', 4.684000e+02)
fuelSalt2.add_nuclide('U234', 2.217000e+06)
fuelSalt2.add_nuclide('U235', 5.092000e+05)
fuelSalt2.add_nuclide('U236', 3.646000e+05)
fuelSalt2.add_nuclide('U237', 4.293000e+02)
fuelSalt2.add_nuclide('U238', 2.872000e+02)
fuelSalt2.add_nuclide('Np237', 7.521000e+04)
fuelSalt2.add_nuclide('Pu238', 6.278000e+04)
fuelSalt2.add_nuclide('Pu239', 1.898000e+04)
fuelSalt2.add_nuclide('Pu240', 7.248000e+03)
fuelSalt2.add_nuclide('Pu241', 1.059000e+03)
fuelSalt2.add_nuclide('Pu242', 2.738000e+02)
fuelSalt2.add_nuclide('Am241', 1.665000e+02)
fuelSalt2.add_nuclide('Am243', 4.781000e+01)
# fuelSalt2.add_nuclide('Sr88', 1.970000e+03)
# fuelSalt2.add_nuclide('Sr89', 9.961000e+02)
# fuelSalt2.add_nuclide('Sr90', 4.124000e+04)
# fuelSalt2.add_nuclide('Y91', 2.053000e+03)
# fuelSalt2.add_nuclide('Sn126', 1.370000e+03)
# fuelSalt2.add_nuclide('Te130', 1.346000e+03)
# fuelSalt2.add_nuclide('Cs137', 1.821000e+04)
# fuelSalt2.add_nuclide('Ba136', 1.794000e+03)
# fuelSalt2.add_nuclide('Ba137', 1.379000e+03)
# fuelSalt2.add_nuclide('Ba138', 2.577000e+04)
# fuelSalt2.add_nuclide('Ba140', 1.826000e+03)
# fuelSalt2.add_nuclide('La139', 3.703000e+03)
# fuelSalt2.add_nuclide('Ce140', 4.737000e+03)
# fuelSalt2.add_nuclide('Ce141', 2.903000e+03)
# fuelSalt2.add_nuclide('Ce142', 5.663000e+03)
# fuelSalt2.add_nuclide('Ce143', 1.999000e+02)
# fuelSalt2.add_nuclide('Ce144', 3.812000e+03)
# fuelSalt2.add_nuclide('Pr141', 3.099000e+03)
# fuelSalt2.add_nuclide('Pr143', 1.352000e+03)
# fuelSalt2.add_nuclide('Nd143', 3.655000e+03)
# fuelSalt2.add_nuclide('Nd145', 3.050000e+03)
# fuelSalt2.add_nuclide('Nd146', 2.418000e+03)
# fuelSalt2.add_nuclide('Nd148', 1.272000e+03)
# fuelSalt2.add_nuclide('Pm147', 1.287000e+03)
fuelSalt2.add_s_alpha_beta('c_Li_in_LiF')
fuelSalt2.add_s_alpha_beta('c_F_in_LiF')
fuelSalt2.temperature = 1021

fuelSalt3 = openmc.Material(1002, "hex")
fuelSalt3.set_density('g/cm3', 4.163)
fuelSalt3.add_nuclide('Li6', 1.135000e+01)
fuelSalt3.add_nuclide('Li7', 4.702000e+06)
fuelSalt3.add_nuclide('F19', 2.752000e+07)
fuelSalt3.add_nuclide('Th232', 3.686000e+07)
fuelSalt3.add_nuclide('U233', 5.025000e+06)
fuelSalt3.add_nuclide('Th230', 8.374000e+02)
fuelSalt3.add_nuclide('Pa233', 2.425000e+04)
fuelSalt3.add_nuclide('U232', 4.684000e+02)
fuelSalt3.add_nuclide('U234', 2.217000e+06)
fuelSalt3.add_nuclide('U235', 5.092000e+05)
fuelSalt3.add_nuclide('U236', 3.646000e+05)
fuelSalt3.add_nuclide('U237', 4.293000e+02)
fuelSalt3.add_nuclide('U238', 2.872000e+02)
fuelSalt3.add_nuclide('Np237', 7.521000e+04)
fuelSalt3.add_nuclide('Pu238', 6.278000e+04)
fuelSalt3.add_nuclide('Pu239', 1.898000e+04)
fuelSalt3.add_nuclide('Pu240', 7.248000e+03)
fuelSalt3.add_nuclide('Pu241', 1.059000e+03)
fuelSalt3.add_nuclide('Pu242', 2.738000e+02)
fuelSalt3.add_nuclide('Am241', 1.665000e+02)
fuelSalt3.add_nuclide('Am243', 4.781000e+01)
# fuelSalt3.add_nuclide('Sr88', 1.970000e+03)
# fuelSalt3.add_nuclide('Sr89', 9.961000e+02)
# fuelSalt3.add_nuclide('Sr90', 4.124000e+04)
# fuelSalt3.add_nuclide('Y91', 2.053000e+03)
# fuelSalt3.add_nuclide('Sn126', 1.370000e+03)
# fuelSalt3.add_nuclide('Te130', 1.346000e+03)
# fuelSalt3.add_nuclide('Cs137', 1.821000e+04)
# fuelSalt3.add_nuclide('Ba136', 1.794000e+03)
# fuelSalt3.add_nuclide('Ba137', 1.379000e+03)
# fuelSalt3.add_nuclide('Ba138', 2.577000e+04)
# fuelSalt3.add_nuclide('Ba140', 1.826000e+03)
# fuelSalt3.add_nuclide('La139', 3.703000e+03)
# fuelSalt3.add_nuclide('Ce140', 4.737000e+03)
# fuelSalt3.add_nuclide('Ce141', 2.903000e+03)
# fuelSalt3.add_nuclide('Ce142', 5.663000e+03)
# fuelSalt3.add_nuclide('Ce143', 1.999000e+02)
# fuelSalt3.add_nuclide('Ce144', 3.812000e+03)
# fuelSalt3.add_nuclide('Pr141', 3.099000e+03)
# fuelSalt3.add_nuclide('Pr143', 1.352000e+03)
# fuelSalt3.add_nuclide('Nd143', 3.655000e+03)
# fuelSalt3.add_nuclide('Nd145', 3.050000e+03)
# fuelSalt3.add_nuclide('Nd146', 2.418000e+03)
# fuelSalt3.add_nuclide('Nd148', 1.272000e+03)
# fuelSalt3.add_nuclide('Pm147', 1.287000e+03)
fuelSalt3.add_s_alpha_beta('c_Li_in_LiF')
fuelSalt3.add_s_alpha_beta('c_F_in_LiF')
fuelSalt3.temperature = 929.4

fuelSalt4 = openmc.Material(1003, "bottom_pipe")
fuelSalt4.set_density('g/cm3', 4.171)
fuelSalt4.add_nuclide('Li6', 1.135000e+01)
fuelSalt4.add_nuclide('Li7', 4.702000e+06)
fuelSalt4.add_nuclide('F19', 2.752000e+07)
fuelSalt4.add_nuclide('Th232', 3.686000e+07)
fuelSalt4.add_nuclide('U233', 5.025000e+06)
fuelSalt4.add_nuclide('Th230', 8.374000e+02)
fuelSalt4.add_nuclide('Pa233', 2.425000e+04)
fuelSalt4.add_nuclide('U232', 4.684000e+02)
fuelSalt4.add_nuclide('U234', 2.217000e+06)
fuelSalt4.add_nuclide('U235', 5.092000e+05)
fuelSalt4.add_nuclide('U236', 3.646000e+05)
fuelSalt4.add_nuclide('U237', 4.293000e+02)
fuelSalt4.add_nuclide('U238', 2.872000e+02)
fuelSalt4.add_nuclide('Np237', 7.521000e+04)
fuelSalt4.add_nuclide('Pu238', 6.278000e+04)
fuelSalt4.add_nuclide('Pu239', 1.898000e+04)
fuelSalt4.add_nuclide('Pu240', 7.248000e+03)
fuelSalt4.add_nuclide('Pu241', 1.059000e+03)
fuelSalt4.add_nuclide('Pu242', 2.738000e+02)
fuelSalt4.add_nuclide('Am241', 1.665000e+02)
fuelSalt4.add_nuclide('Am243', 4.781000e+01)
# fuelSalt4.add_nuclide('Sr88', 1.970000e+03)
# fuelSalt4.add_nuclide('Sr89', 9.961000e+02)
# fuelSalt4.add_nuclide('Sr90', 4.124000e+04)
# fuelSalt4.add_nuclide('Y91', 2.053000e+03)
# fuelSalt4.add_nuclide('Sn126', 1.370000e+03)
# fuelSalt4.add_nuclide('Te130', 1.346000e+03)
# fuelSalt4.add_nuclide('Cs137', 1.821000e+04)
# fuelSalt4.add_nuclide('Ba136', 1.794000e+03)
# fuelSalt4.add_nuclide('Ba137', 1.379000e+03)
# fuelSalt4.add_nuclide('Ba138', 2.577000e+04)
# fuelSalt4.add_nuclide('Ba140', 1.826000e+03)
# fuelSalt4.add_nuclide('La139', 3.703000e+03)
# fuelSalt4.add_nuclide('Ce140', 4.737000e+03)
# fuelSalt4.add_nuclide('Ce141', 2.903000e+03)
# fuelSalt4.add_nuclide('Ce142', 5.663000e+03)
# fuelSalt4.add_nuclide('Ce143', 1.999000e+02)
# fuelSalt4.add_nuclide('Ce144', 3.812000e+03)
# fuelSalt4.add_nuclide('Pr141', 3.099000e+03)
# fuelSalt4.add_nuclide('Pr143', 1.352000e+03)
# fuelSalt4.add_nuclide('Nd143', 3.655000e+03)
# fuelSalt4.add_nuclide('Nd145', 3.050000e+03)
# fuelSalt4.add_nuclide('Nd146', 2.418000e+03)
# fuelSalt4.add_nuclide('Nd148', 1.272000e+03)
# fuelSalt4.add_nuclide('Pm147', 1.287000e+03)
fuelSalt4.add_s_alpha_beta('c_Li_in_LiF')
fuelSalt4.add_s_alpha_beta('c_F_in_LiF')
fuelSalt4.temperature = 920.6

boronCarbide = openmc.Material(1004, "absorber")
boronCarbide.set_density('g/cm3', 2.52016)
boronCarbide.add_element('B', 4.0)
boronCarbide.add_element('C', 1.0)
boronCarbide.temperature = 975

alloy = openmc.Material(1005, "reflector")
alloy.set_density('g/cm3', 10)
alloy.add_element('Ni', 79.432)
alloy.add_element('W', 9.976)
alloy.add_element('Cr', 8.014)
alloy.add_element('Mo', 0.736)
alloy.add_element('Fe', 0.632)
alloy.add_element('Ti', 0.295)
alloy.add_element('C', 0.294)
alloy.add_element('Mn', 0.257)
alloy.add_element('Si', 0.252)
alloy.add_element('Al', 0.052)
alloy.add_element('B', 0.033)
alloy.add_element('P', 0.023)
alloy.add_element('S', 0.004)
alloy.temperature = 975

materials_file = openmc.Materials([fuelSalt1, fuelSalt2, fuelSalt3, fuelSalt4, boronCarbide, alloy])

blanketSalt = []
i = 0
while i < 33:
    x = openmc.Material(i+1, "blanket{0}".format(i+1))
    blanketSalt.append(x)
    blanketSalt[i].add_nuclide('Li6', 7.75e-4)
    blanketSalt[i].add_nuclide('Li7', 77.499225)
    blanketSalt[i].add_nuclide('F19', 167.5)
    blanketSalt[i].add_nuclide('Th232', 22.5)
    blanketSalt[i].add_s_alpha_beta('c_Li_in_LiF')
    blanketSalt[i].add_s_alpha_beta('c_F_in_LiF')
    blanketSalt[i].set_density('g/cm3', 4.123)
    blanketSalt[i].temperature = 975
    materials_file.append(blanketSalt[i])
    i = i + 1

materials_file.export_to_xml()


# define settings
settings = openmc.Settings()
point = openmc.stats.Point((75, 0 ,0))
src = openmc.Source(space=point)
settings.source = src
settings.dagmc = True
settings.particles = 50000
settings.batches = 300
settings.inactive = 50
settings.output = {'tallies':False}
settings.resonance_scattering = {"enable":True, "method":"dbrc"}
settings.temperature = {"multipole":False, "method":"interpolation"}
settings.export_to_xml()

openmc.run()
```

**Comment by pshriwise** [comment]
Thanks for sharing this, @D-Benjamin! Is it possible to share the DagMC model as well?

**Comment by D-Benjamin** [comment]
> Thanks for sharing this, @D-Benjamin! Is it possible to share the DagMC model as well?

Yes, but here can not upload the DAGMC model, can you give me your Gmail address? Thanks!

**Comment by pshriwise** [comment]
Can you please upload the model here? https://tinyurl.com/yxvnmj7v

**Comment by pshriwise** [comment]
I managed to reproduce this error @D-Benjamin. Thank you for reporting it! 

The problem seems to be entirely related to creating a material with an ID of 0. I've opened PR #1367 to address this.

**Comment by D-Benjamin** [comment]
> I managed to reproduce this error @D-Benjamin. Thank you for reporting it!
> 
> The problem seems to be entirely related to creating a material with an ID of 0. I've opened PR #1367 to address this if you want to see more details.

Thanks, @pshriwise . I also have another issue. After running "openmc", there is a warnning when reading cross sections:
```
WARNING: Negative value(s) found on probability table for nuclide I131 at 920K
 WARNING: Negative value(s) found on probability table for nuclide I131 at 940K
 WARNING: Negative value(s) found on probability table for nuclide I131 at 960K
 WARNING: Negative value(s) found on probability table for nuclide I131 at 980K
 WARNING: Negative value(s) found on probability table for nuclide I131 at 1020K
 WARNING: Negative value(s) found on probability table for nuclide I131 at 1040K

WARNING: Negative value(s) found on probability table for nuclide Sn123 at 920K
 WARNING: Negative value(s) found on probability table for nuclide Sn123 at 940K
 WARNING: Negative value(s) found on probability table for nuclide Sn123 at 960K
 WARNING: Negative value(s) found on probability table for nuclide Sn123 at 980K
 WARNING: Negative value(s) found on probability table for nuclide Sn123 at
          1020K
 WARNING: Negative value(s) found on probability table for nuclide Sn123 at
          1040K
```

The cross sections were processed by NJOY2016 like that:
```
url = 'https://t2.lanl.gov/nis/data/data/ENDFB-VII.1-neutron/I/131'
filename, headers = urllib.request.urlretrieve(url, 'I131.endf')
I131 = openmc.data.IncidentNeutron.from_njoy('I131.endf', temperatures=tem, stdout=True)
I131.export_to_hdf5('I131.h5', 'w')
```
Except Sn123 and I131, others didn't report this WARNING, what's wrong with them?

Best Regards!

**Comment by pshriwise** [comment]
I'm not really sure why you'd be seeing that. Were there any errors/warnings during the conversion to our hdf5 format?

**Comment by paulromano** [comment]
Because probability tables are generated using a sequence of resonances represented the single-level Breit-Wigner (SLBW) formalism, it's theoretically possible to have negative elastic scattering cross sections. NJOY does not protect against this and so sometimes you do see negative values in a probability table. At runtime, if OpenMC encounters negative cross sections as a result of this, it sets them to zero.

**Comment by paulromano** [comment]
To summarize -- the original issue had to do with a variable not getting initialized properly, which was fixed in #1367.
